src/test/resources/testData.json

```diff
@@ -7,6 +7,28 @@
       "password": "eotTesting@0",
       "firstName": "Eot",
       "lastName": "Testing"
+    },
+    "Guest": {
+      "indigoGiftVoucher": {
+        "denominations": [
+          3000,
+          5000,
+          10000
+        ],
+        "quantities": [
+          1,
+          2,
+          3,
+          4,
+          5
+        ],
+        "promoCodes": [
+          "INVALID_PROMO1",
+          "INVALID_PROMO2",
```

Comment:
Use testdata.json file as less as possible, promo code should be random - generated at runtime